data/json/npcs/talk_tags.json

```
@@ -242,7 +288,15 @@
       "you're a dead man",
       "you'll taste my <mywp>",
       "you're <swear> dead",
-      "<name_b>, <ill_kill_you>"
+      "<name_b>, <ill_kill_you>",
+      "Only one of us is leaving here alive.",
```

```suggestion
      "only one of us is leaving here alive",
```
Format for the snippets in this category is no caps, no punctuation.